packages/eslint-plugin-template/src/rules/conditional-complexity.ts

```diff
@@ -21,37 +25,41 @@ export default createESLintRule<Options, MessageIds>({
   meta: {
     type: 'suggestion',
     docs: {
-      description: `The condition complexity shouldn't exceed a rational limit in a template.`,
+      description:
+        'The conditional complexity should not exceed a rational limit',
       category: 'Best Practices',
       recommended: false,
     },
     schema: [
       {
         type: 'object',
-        properties: {
-          maxComplexity: {
-            type: 'number',
-            minimum: 1,
-          },
-        },
+        properties: { maxComplexity: { minimum: 1, type: 'number' } },
         additionalProperties: false,
       },
     ],
     messages: {
       conditionalСomplexity:
-        'The condition complexity "{{totalComplexity}}" exceeds the defined limit "{{maxComplexity}}"',
+        'The conditional complexity "{{totalComplexity}}" exceeds the defined limit "{{maxComplexity}}"',
     },
   },
   defaultOptions: [{ maxComplexity: 5 }],
   create(context, [{ maxComplexity }]) {
+    ensureTemplateParser(context);
```

Comment:
Is this still necessary since I'm using `parserServices` (for the `BoundAttribute` case)?

Reply:
No, you're correct, it is only necessary if not using `parserServices`, just helps protect users from leveraging template rules with the wrong parser in their configs